usagov/vote-gov-nvrf-app
# Vote-320 -> Adding Cypress Tests

Ticket: [Vote-320](https://cm-jira.usa.gov/browse/VOTE-320)
Purpose: This Pr will add the fist pass at setting up and adding cypress testing to the NVRF project 
to test:

1. pull down branch 
2. cd into `testing` and run npm install 
3. run `npm run cy:open` and check that the runner is working and tests are passing 
4. run `npm run cy:test` and check that the tests are running headless 

**More To Do**
There is more work to be done on the app but will come back to build on this work and add more validation.
